components/SkilledImpact/elements/ApplicationContent.js

```diff
@@ -18,6 +18,15 @@ class ApplicationContent extends React.Component {
       selectedExpanded: true,
       notSelectedExpanded: true,
     };
+    this.noDecisionSubmissions = props.submissions.map(
+      (submission) => submission.submission.decision === 'PENDING'
+    );
+    this.selectedSubmissions = props.submissions.map(
+      (submission) => submission.submission.decision === 'ACCEPTED'
+    );
+    this.notSelectedSubmissions = props.submissions.map(
+      (submission) => submission.submission.decision === 'DENIED'
+    );
```

Comment:
These should be using `.filter()`, not `.map()`, right?

Reply:
oh yeah whoops, thanks for the catch